### PR TITLE
Refactor static analyser

### PR DESCRIPTION
**Pull request by tizah**
Setting up and running the project

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "prettier.semi": false
+}
```
Comment (LianaHus) [inline .vscode/settings.json:3]: new line

File: apps/remix-ide/src/app/panels/file-panel.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+import { Workspace } from '@remix-ui/workspace'; // eslint-disable-line
+import * as ethutil from 'ethereumjs-util';
+import { checkSpecialChars, checkSlash } from '../../lib/helper';
+var EventManager = require('../../lib/events');
+var { RemixdHandle } = require('../files/remixd-handle.js');
+var { GitHandle } = require('../files/git-handle.js');
+var globalRegistry = require('../../global/registry');
+var examples = require('../editor/examples');
+var GistHandler = require('../../lib/gist-handler');
+var QueryParams = require('../../lib/query-params');
```
Comment (LianaHus) [inline apps/remix-ide/src/app/panels/file-panel.js:15]: no need for ';'
some linters are not allowing them

File: apps/remix-ide/src/app/panels/file-panel.js [hunk 3]
```diff
@@ -34,44 +34,56 @@ var QueryParams = require('../../lib/query-params')
...
+    'createWorkspace'
+  ],
   events: ['setWorkspace', 'renameWorkspace', 'deleteWorkspace'],
   icon: 'assets/img/fileManager.webp',
   description: ' - ',
   kind: 'fileexplorer',
   location: 'sidePanel',
-  documentation: 'https://remix-ide.readthedocs.io/en/latest/file_explorer.html',
+  documentation:
+    'https://remix-ide.readthedocs.io/en/latest/file_explorer.html',
```
Comment (LianaHus) [inline apps/remix-ide/src/app/panels/file-panel.js:50]: no need for new line

File: apps/remix-ide/src/app/panels/file-panel.js (hunk 2 above)
Comment (tizah) [reply apps/remix-ide/src/app/panels/file-panel.js:15]: removed semi colons too

**Pull request closed without merging**